api/src/test/java/org/pmiops/workbench/db/dao/UserServiceTest.java

```diff
@@ -107,101 +132,113 @@ public void testSyncComplianceTrainingStatus() throws Exception {
     assertThat(user.getComplianceTrainingExpirationTime()).isEqualTo(new Timestamp(12345));
 
     // Completion timestamp should not change when the method is called again.
-    CLOCK.increment(1000);
+    incrementClock();
```

Comment:
I'm not sure this abstraction gives us much other than obfuscating milliseconds. I could see an argument for pulling it out, but I'd like it to either be configurable or clear how much it increments by. (So either `incrementClockOneThousandMillis` or `incrementClock(Integer millis)`

Reply:
If this were a public API I'd probably agree with you. Private methods (especially in test classes) don't need to be big reusable abstractions. 1000 ms is an internal detail, since the actual value doesn't really matter much for our purposes, so I hid it. I explicitly don't want the reader to have to care how big the increment is in this case, as it's irrelevant to the test. Anyone interested can dive into its implementation and immediately see it. A better name might be `tick()`.

Finally, there's a big difference between Encapsulation and Obfuscation. Encapsulation makes things simpler by hiding private details, whereas Obfuscation makes things more complex or difficult to understand by introducing convoluting details, constraints, and redirections.

Reply:
Talked offline. We agreed that `tick()` might be better. My problem was it added enough of a redirection while not hiding enough private details for me. So it felt like it added a redirect without actually giving us much. (It hides the 1000, but for me, because it was `increment`, I thought the number was relevant)